### PR TITLE
Fix inference of locale table and fix exposed locale bugs

### PR DESCRIPTION
**Pull request by emmericp**
This exposed a few minor bugs in locales, most are likely in effectively unused code. But the raid lead override notification message not working was a real bug :)

### CODE REVIEW

**Review by QartemisT — APPROVED** [review]

File: DBM-Core/DBM-Core.lua [hunk 1]
```diff
@@ -4178,7 +4178,7 @@ do
 			private.chatBubblesDisabled = true
 		end
 		if activated then
-			AddMsg(L.OVERRIDE_ACTIVATED)
+			DBM:AddMsg(L.OVERRIDE_ACTIVATED)
```
Comment (Zidras) [inline DBM-Core/DBM-Core.lua:4181]: @emmericp is this change needed? the function is cached
 https://github.com/DeadlyBossMods/DBM-Unified/blob/a1d0466ea4f80c54eac0b3e1351756948dd5a17d/DBM-Core/DBM-Core.lua#L7023

**Comment by MysticalOS** [comment]
Yeah i wondered same, but there are a few things his checker can't identnify accurately so sometimes he "fixes" things that aren't broken.

File: DBM-Core/DBM-Core.lua (hunk 1 above)
Comment (emmericp) [reply DBM-Core/DBM-Core.lua:4181]: the self param was missing, it's either this or `AddMsg(DBM, L....)`

Comment (emmericp) [reply DBM-Core/DBM-Core.lua:4181]: understood. but then it needs to be AddMsg(DBM, "Foo"), the old code just printed nil.
